src/fluree/db/query/fql.cljc

```diff
@@ -654,59 +650,81 @@
           (assoc res k' (conj (get res k' []) v))))
       {} tuples)))
 
+(defn- build-order-fn
+  [orderBy groupBy]
```

Comment:
This is a minor style suggestion that shouldn't hold up the merge unless you feel like making the changes so take it or leave it.

Idiomatic clojure code typically uses `kebab-case` names instead of `camelCase` ones, so the argument list here would typically be written as `[order-by group-by]` instead. The same goes for the names of the other bindings here. 

I find [The Clojure Style Guide](https://github.com/bbatsov/clojure-style-guide) helpful for these little rules. I usually have it open in a tab when I'm writing code.